lib/itk/queue/consumer.ex

```diff
@@ -65,6 +65,15 @@ defmodule ITKQueue.Consumer do
   end
 
   defp consume(channel, meta = %{delivery_tag: tag, headers: headers}, payload, subscription = %Subscription{queue_name: queue_name, routing_key: routing_key, handler: handler}) do
+    message_uuid = UUID.uuid4()
```

Comment:
One step closer to passing these around and being able to track data across services. 👍